src/modules/setup/index.ts

```diff
@@ -0,0 +1,13 @@
+import { getManifest } from '../../manifest'
+import { setupESLint } from './setupESLint'
+import { setupTSConfig } from './setupTSConfig'
+import { setupTypings } from './setupTypings'
+
+const buildersToAddAdditionalPackages = ['react', 'node']
+
+export default async () => {
+  const manifest = await getManifest()
+  await setupESLint(manifest, buildersToAddAdditionalPackages)
+  await setupTSConfig(manifest)
+  await setupTypings(manifest)
```

Comment:
Can you do these three steps asynchronously? I think the problem is setting the package.json, which is performed by two of them. But maybe we can have only two process running asynchronously: one updating the package and another setting up the ts config.